src/Bitmap_cubical_complex/benchmark/persistence_2d.cpp

```diff
@@ -0,0 +1,92 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
```

Comment:
Do we move this file also in Persistent_cohomology/benchmark ?

Reply:
Oops... done.